src/Tribe/Views/V2/Views/Month_View.php

```diff
@@ -423,6 +423,46 @@ protected function get_url_date_format() {
 		return 'Y-m';
 	}
 
+	/**
+	 * Creates a HTML link and "fast forward" message to append to the "no events found" message.
+	 *
+	 * @since TBD
+	 *
+	 * @param bool  $canonical         Whether to return the canonical (pretty) version of the URL or not.
+	 * @param array $passthru_vars     An optional array of query variables that should pass thru the method untouched
+	 *                                 in key and value.
+	 * @return string                  The html link and message.
+	 */
+	public function fast_forward( $canonical = false, array $passthru_vars = [] ) {
+		$date      = $this->context->get( 'event_date', $this->context->get( 'today' ) );
+		$cache_key = __METHOD__ . '_' . md5( wp_json_encode( array_merge( [ $date, $canonical ], $passthru_vars ) ) );
+
+		if ( isset( $this->cached_urls[ $cache_key ] ) ) {
+			return $this->cached_urls[ $cache_key ];
+		}
+
+		$next_event = (array) tribe_events()->where( 'starts_after', $date )->per_page( 1 )->fields( 'ids' )->first();
+		$next_event = tribe_get_event( array_shift( $next_event ) );
+
+		if ( empty( $next_event ) ) {
+			return false;
+		}
+
+		$url_date   = Dates::build_date_object( $next_event->start_date );
+		$url        = $this->build_url_for_date( $url_date, $canonical, $passthru_vars );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 6 spaces but found 8 spaces

<sub>hash: d790aa78e69f9cbd6a808defcfa7de19</sub>